Comp2022/src/main/cpp/subsystems/Drivetrain.cpp

```diff
@@ -658,34 +658,27 @@ void Drivetrain::MoveWithLimelightInit(bool m_endAtTarget)
         double ty = robotContainer->m_vision.GetVertOffsetDeg();
         m_limelightDistance = m_slope * ty + m_distOffset;
         m_targetDistance = m_limelightDistance;
+        spdlog::info("DTL Using Current Distance {}", m_targetDistance);
     }
 }
 
 void Drivetrain::MoveWithLimelightExecute(double tx, double ty, bool tv)

```

Comment:
since all the distance calculations are done in vision now, why do we still need these parameters? just to print them out?